amazon.sql

```diff
@@ -0,0 +1,91 @@
+-- MySQL Workbench Forward Engineering
+
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION';
+
+-- -----------------------------------------------------
+-- Schema mydb
+-- -----------------------------------------------------
+-- -----------------------------------------------------
+-- Schema amazon
+-- -----------------------------------------------------
+
+-- -----------------------------------------------------
+-- Schema amazon
+-- -----------------------------------------------------
+CREATE SCHEMA IF NOT EXISTS `amazon` DEFAULT CHARACTER SET latin1 ;
+USE `amazon` ;
+
+-- -----------------------------------------------------
+-- Table `amazon`.`product`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `amazon`.`product` (
+  `id` INT(11) NOT NULL,
+  `title` VARCHAR(50) NOT NULL,
+  `description` VARCHAR(200) NOT NULL,
+  `rating` INT(11) NULL DEFAULT NULL,
+  `available` TINYINT(4) NOT NULL,
+  `deliverytime` INT(11) NOT NULL,
+  `Imageurl` VARCHAR(200) NOT NULL,
```

Comment:
use either all small or all capital letters, and use underscore if there are two words